.github/pr_instructions.md

```diff
@@ -0,0 +1,67 @@
+3. [Create a branch.](create-a-branch)
+4. [Prepare the dataset.](prepare-the-dataset)
+
+## Find a dataset
+
+Find a dataset that would be good for TidyTuesday: either one that is already ready for analysis, or one that you can clean so that it meets the criteria. 
+These are the requirements for a dataset:
+- Data can be saved as one or more CSV files.
+- The whole dataset (all files) is less than 20MB.
+- You can describe each variable (either using an existing data dictionary or by creating your own dictionary).
+- The data is publicly available and free for reuse, either with or without attribution.
+
+You will also need:
+- The source of the dataset
+- An article about the dataset or that uses the dataset
+- At least one image related to or using the dataset
+
+## Prepare your repository
+
+You'll need to perform this step the first time you submit a pull request to this repository. A "pull request" is a submission of code to a git repository. 
+If you have never worked with git before, that's fine! We'll help you get set up.
+
+1. Set up git, github, and your IDE (such as RStudio). We have step-by-step [instructions for setting up things to work with the Data Science Learning Community](https://github.com/r4ds/bookclub-setup?tab=readme-ov-file#setting-up-for-data-science-learning-community-book-clubs).
+2. Fork the tidytuesday repository (this one). In R, you can use `usethis::create_from_github("rfordatascience/tidytuesday")` to create your personal fork on GitHub and copy it to your computer. Note: This step is currently quite slow, ad requires about **8 GB** of space on disk.
+
+## Create a branch
+
+We use a fork/branch approach to pull requests, meaning you'll create a version of the repo specifically for your changes, and then ask us to merge those changes into the main tidytuesday repository.
+
+1. If you are on anything other than the `master` branch of your local repository, switch back to master. This might happen if you submitted a previous dataset, for example. In R, you can use `usethis::pr_pause()` (if your previous submission is still pending), or `usethis::pr_finish()` (if we've accepted your submission).
+2. Pull the latest version of the repository to your computer. In R, use `usethis::pr_merge_main()`
+3. Create a new branch, with something similar to the name of the dataset you're submitting. For instance if it's a dataset on American baseball, something like "american-baseball"" or "baseball" works. In R, you can create this branch using `usethis::pr_init(BRANCHNAME)`, eg `usethis::pr_init("american-baseball")`.
+4. Navigate to the `data/curated` folder in your branch of the repository. 
+5. Make a copy of the `template` folder for your dataset, inside the `curated` folder. Name it something descriptive, like "funspotr" or "ttmeta", not "my_dataset". We might accept your dataset without assigning it to a specific week yet, in which case this folder will need to be unique.
```

Comment:
I think removing the  reference to "funspotr" or "ttmeta" would be helpful for folks who have not participated in previous Tidy Tuesdays. I know what "ttmeta" is but have no idea what's "funspotr", so it's confusing unless you provide more context to what those describe. I think just saying a "short descriptive name no more than ___ characters" may suffice.